src/Horse.ISAPI.pas

```diff
@@ -0,0 +1,28 @@
+unit Horse.ISAPI;
+
+interface
+
+uses Horse.API;
+
```

Comment:
Added shortcut in uses of THorseRequest and THorseResponse, same as Horse.Core file.